### ISSUE TITLE
Create component to manage secondary charts

### ISSUE THREAD

**Issue by jleft** [ScottLogic/d3fc-showcase#331]

**Comment by jdhodges** [comment]
Is the idea here to have the secondaries grouped in a container?

**Comment by jleft** [comment]
Yes. Rather than having [three containers permanently](https://github.com/ScottLogic/BitFlux/blob/develop/src/assets/js/app.js#L34-L36) in the page and having app do [lots](https://github.com/ScottLogic/BitFlux/blob/develop/src/assets/js/app.js#L114-L121) [of](https://github.com/ScottLogic/BitFlux/blob/develop/src/assets/js/app.js#L379-L382) [logic](https://github.com/ScottLogic/BitFlux/blob/develop/src/assets/js/app.js#L385-L386) for the secondaries, the idea is to have a component which can manage this. (Complexity in layout.js should also be reduced.)

I expect it would be similar to how we use multi/Cartesian chart with the series/indicators, but rather than the renderers being rendered into the same container, each renderer would have its own container. It might look something like this:

```js
var secondaries = multiChart() // <-- new component
    .charts([rsi, volume]);

secondariesContainer.call(secondaries);
```

Give me a shout if you'd like to chat through this :)
